kview/src/main/java/dev/ikm/komet/kview/lidr/mvvm/view/details/LidrDetailsController.java

```diff
@@ -353,6 +356,7 @@ public void initialize() {
         if (propertiesToggleButton.isSelected() || isOpen(propertiesSlideoutTrayPane)) {
             updateDraggableNodesForPropertiesPanel(true);
         }
+        popupStampButton.disableProperty().set(lidrViewModel.getPropertyValue(MODE) == EDIT);
```

Comment:
Can you create a binding in the initialize() method?
```java
StringProperty mode = lidrViewModel.getStringProperty(MODE);
BooleanBinding isEditMode = Bindings.createBooleanBinding(
            () -> mode.get() != null && mode.get().equalsIgnoreCase(MODE),
            mode
        );
popupStampButton.disableProperty().bind(isEditMode);
```
I believe this allows windows that were in 'Create' mode, such as the Concept window, will subsequently become in 'Edit' mode. Create concept [create mode] -> becomes [edit mode]. The button stamp will disable automatically.